<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="mkrishnan" name="com.avlview.app.testcases.ClientsPageTest" tests="7" failures="1" timestamp="09 Jan 2019 10:24:51 GMT" time="176.104" errors="0">
  <testcase name="validateclientpageExistTest" time="21.317" classname="com.avlview.app.testcases.ClientsPageTest"/>
  <testcase name="validatePartnerLogoExistTest" time="20.413" classname="com.avlview.app.testcases.ClientsPageTest"/>
  <testcase name="validateAddclientExistTest" time="19.590" classname="com.avlview.app.testcases.ClientsPageTest"/>
  <testcase name="validateSearchExistTest" time="18.648" classname="com.avlview.app.testcases.ClientsPageTest"/>
  <testcase name="validateSearchResultTest" time="26.138" classname="com.avlview.app.testcases.ClientsPageTest"/>
  <testcase name="validateTotalTabsTest" time="24.264" classname="com.avlview.app.testcases.ClientsPageTest">
    <failure type="java.lang.AssertionError" message="expected [2] but found [3]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [3]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.avlview.app.testcases.ClientsPageTest.validateTotalTabsTest(ClientsPageTest.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateTotalTabsTest -->
  <testcase name="validateRemoteLoginTest" time="45.734" classname="com.avlview.app.testcases.ClientsPageTest"/>
</testsuite> <!-- com.avlview.app.testcases.ClientsPageTest -->
